packages/hawtio/src/util/objects.ts

```diff
@@ -12,7 +12,7 @@ export function isEmpty(obj: object): boolean {
 }
 
 export function isString(value: unknown): value is string {
-  return typeof value === 'string' || value instanceof String
+  return value != null && typeof value.valueOf() === 'string'
```

Comment:
Why not just typeof value?

Reply:
because it not only handles `string` but also `String` objects.